app/assets/stylesheets/rooms.scss

```diff
@@ -108,3 +108,9 @@
     opacity: 0.6;
   }
 }
+
+#rooms-list-empty {
+  svg {
+    padding-top: 2rem;
+  }
+}
```

Comment:
This can be done in bootstrap

Reply:
It can't, pt-4 is 1.5rem and pt-5 is 3rem.